apps/src/weblab/CdoBramble.js

```diff
@@ -10,12 +10,15 @@ import firehoseClient from '../lib/util/firehose';
 
 // Some temporary logging to diagnose possible loading issues.
 function tempLog(event, data = null) {
+  const dataJson = {
+    is_cros: /\bCrOS\b/.test(navigator.userAgent),
```

Comment:
I think we might actually get the user agent separately, so this was done as a convenience, but might not be necessary.

Reply:
Oh yep you're right - this shows up in the "device" payload.